### PR TITLE
Update README with information on open CVE

### PR DESCRIPTION
**Pull request by joshcanhelp**

### CODE REVIEW

**Review by lbalmaceda — COMMENTED** [review]
@gkwang are you happy with this change?

**Comment by gkwang** [comment]
yes,  let's suppress the vulnerability as well since it doesn't affect us.

**Comment by joshcanhelp** [comment]
@lbalmaceda - Is that something you can do? I'm guessing this would need to be force-merged if not.

**Review by lbalmaceda — APPROVED** [review]

**Comment by lbalmaceda** [comment]
@joshcanhelp I've ignored the issue until 2020y. We can't ignore it forever.. but we can check in a few months if another solution arises or we can re-ignore.

File: README.md [hunk 1]
```diff
@@ -31,6 +33,12 @@ Add the following line to your `Gemfile`:
 gem 'omniauth-auth0'
 ```
 
+If you're using this strategy with Rails, also add the following for CSRF protection:
+
+```ruby
+gem 'omniauth-rails_csrf_protection'
```
Comment (jjf21) [inline README.md:39]: Do we still need to add this gem?
It's written on the [gem's](https://github.com/cookpad/omniauth-rails_csrf_protection) readme:

Image

**Pull request merged**